ui/components/app/smart-transactions/smart-transactions-opt-in-modal.tsx

```diff
@@ -89,7 +91,7 @@ const NoThanksLink = ({
       width={BlockSize.Full}
       className="mm-smart-transactions-opt-in-modal__no-thanks-link"
     >
-      {t('dontEnableEnhancedProtection')}
+      {t('manageInSettings')}
```

Comment:
Dan, I was thinking about the "managing in settings" button and had a small suggestion. The button text seems to imply that the user is deferring the opt-in decision rather than explicitly declining it. Since the default for an "opt-in" is false, calling `setSmartTransactionsOptInStatus(false)` might not be necessary when they click that button.

However, if we need to track whether the user has seen the opt-in modal (set smartTransactionsOptInStatus to something other than undefined), maybe we could rename the handler to "handleMarkOptInModalSeen" instead of "handleNoThanks"? And we could add a comment like: "We `setSmartTransactionsOptInStatus(false)` to track that user has seen the opt-in modal."

What do you think?

Reply:
As we have discussed, I've renamed the `NoThanks` component to `ManageInSettings` + its on-click handler as well. I've also added an explanation comment about that we need to call `setSmartTransactionsOptInStatus(false)`, so a user wouldn't see the opt-in modal again.